service_capacity_modeling/interface.py

```diff
@@ -680,11 +680,13 @@ class CurrentClusterCapacity(ExcludeUnsetModel):
     cluster_instance: Optional[Instance] = None
     cluster_instance_count: Interval
     # The distribution cpu utilization in the cluster.
-    cpu_utilization: Interval
+    cpu_utilization: Interval = certain_float(0.0)
```

Comment:
Would this be better as Optional[Interval]?

Reply:
I was debating it. Optional is a little annoying with having to be checked when reading it. I was thinking the defaults could be used safely.